Buildings/Controls/OBC/CDL/Logical/Sources/SampleTrigger.mo

```diff
@@ -2,13 +2,22 @@ within Buildings.Controls.OBC.CDL.Logical.Sources;
 block SampleTrigger "Generate sample trigger signal"
   parameter Modelica.SIunits.Time period(
     final min=Constants.small) "Sample period";
```

Comment:
Should `Buildings/Controls/OBC/CDL/Integers/Sources/TimeTable.mo` line 11 also be `Constants.small`?